doc/conf.py

```diff
@@ -22,6 +22,9 @@
 # is relative to the documentation root, use os.path.abspath to make it
 # absolute, like shown here.
 sys.path.insert(0, os.path.abspath('sphinxext'))
+# Path for sphinx_copybutton ext.
+sys.path.insert(0, os.path.abspath(
```

Comment:
This should not be necessary if there is a file `sphinxext/sphinx_copybutton/__init__.py`